app/models/post.rb

```diff
@@ -2,9 +2,11 @@ class Post < ApplicationRecord
   validates_presence_of :type, :permalink_url, :fid, :user
   validates_uniqueness_of :permalink_url, :fid
 
-  belongs_to :user, counter_cache: true
-  has_many :comments
-  has_many :reactions
+  belongs_to :user,    counter_cache: true
+  has_many :comments,  dependent: :destroy
+  has_many :reactions, dependent: :destroy
+  has_many :post_tags, dependent: :destroy
+  has_many :tags,      through: :post_tags
```

Comment:
humm, can we give up the vertical alignment? it's unnecessary

Reply:
yeah, of course we can

![](https://media0.giphy.com/media/2WxWfiavndgcM/200.gif)